app/src/main/java/com/safetyheads/akademiaandroida/fragments/technologystack/TechnologyStackTab.kt

```diff
@@ -0,0 +1,5 @@
+package com.safetyheads.akademiaandroida.fragments.technologystack
+
+enum class TechnologyStackTab {
```

Comment:
In the future we will base on String, see: https://trello.com/c/DkPOTMu4/117-pobieranie-danych-z-firebase-firestore-technologystack